From dd29f173c44315b921a60e115537cf62e6d0b332 Mon Sep 17 00:00:00 2001
From: Chunhe Lan <Chunhe.Lan@freescale.com>
Date: Wed, 30 Apr 2014 16:14:20 +0800
Subject: [PATCH 664/987] t4240rdb/dts: Fix mac address error of fm2-mac9 and
 fm2-mac10 port

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

According to convention, 10Gbps SFP+ ports of fman2 should use
eth14 address and eth15 address, and not eth4 address and eth5
address.

Signed-off-by: Chunhe Lan <Chunhe.Lan@freescale.com>
Change-Id: I8e85db7bdb2c06a3883e3d474e6d306371352ecc
Reviewed-on: http://git.am.freescale.net:8181/11702
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Shaohui Xie <Shaohui.Xie@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
---
 arch/powerpc/boot/dts/t4240rdb.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/powerpc/boot/dts/t4240rdb.dts b/arch/powerpc/boot/dts/t4240rdb.dts
index 6836aff1..b1f3997 100644
--- a/arch/powerpc/boot/dts/t4240rdb.dts
+++ b/arch/powerpc/boot/dts/t4240rdb.dts
@@ -54,8 +54,8 @@
 		ethernet1 = &enet1;
 		ethernet2 = &enet2;
 		ethernet3 = &enet3;
-		ethernet4 = &enet14;
-		ethernet5 = &enet15;
+		ethernet4 = &enet4;
+		ethernet5 = &enet5;
 		ethernet6 = &enet6;
 		ethernet7 = &enet7;
 		ethernet8 = &enet8;
@@ -64,8 +64,8 @@
 		ethernet11 = &enet11;
 		ethernet12 = &enet12;
 		ethernet13 = &enet13;
-		ethernet14 = &enet4;
-		ethernet15 = &enet5;
+		ethernet14 = &enet14;
+		ethernet15 = &enet15;
 	};
 
 	ifc: localbus@ffe124000 {
-- 
1.9.1

